pkg/storaged/mount-users.py

```diff
@@ -21,10 +21,12 @@ def fuser(mount_point):
     results = {}
 
     def get_cmdline(pid):
-        return " ".join(open("/proc/%s/cmdline" % pid).read().split("\0"))
+        with open(f"/proc/{pid}/cmdline") as f:
+            return " ".join(f.read().split("\0"))
```

Comment:
This screams out for `shlex.join()` or `map(shlex.quote, ...)`.

Reply:
Interestingly this does the  wrong thing with the trailing `\0`, as it ends up as an extra `''` argument. This works, though:

    shlex.join(f.read().rstrip("\0").split("\0"))

So going use that. Added as an extra commit as it's a behaviour change, and unrelated to opening files.